exe/Wrapper.hs

```diff
@@ -79,6 +80,10 @@ main = do
           putStrLn hlsVer
           putStrLn "Tool versions found on the $PATH"
           putStrLn $ showProgramVersionOfInterest programsOfInterest
+          putStrLn "Tool versions found by cradle"
+          cradle <- findProjectCradle' False
+          ghcVersion <- runExceptT $ getRuntimeGhcVersion' cradle
+          putStrLn $ "ghc:\t\t" ++ fromRight "Not found" ghcVersion
```

Comment:
While you are at it, would you mind replacing `\t\t` with always the same amount of spaces (e.g., in Version.hs, too)? The tabulator doesn't render as I initially intended and it aligns baldy on some systems.

Reply:
Would you prefer manually adding a handful of spaces for each line (depending on the string length) or can we drop indentation and use a single space everywhere?

Reply:
I think it would be nicer if it was aligned, but I leave it up to you, since you are doing me the favour :)

Reply:
Sure, changed it a bit to make sure the number of spaces is the same amount in both places: 6cfb60820ce25480310a34c7ca70546ce95f4076